packages/typography/src/Body/Body.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { type StoryMetaType } from '@lg-tools/storybook-utils';
+
+import Body from './Body';
+import { BodyProps } from './Body.types';
+
+export const Basic = ({ children, ...rest }: BodyProps) => {
+  return <Body {...rest}>{children}</Body>;
+};
+
+const meta: StoryMetaType<typeof Body> = {
+  title: 'Components/Typography/Body',
+  component: Body,
+  parameters: {
+    default: 'Basic',
+    generate: {
+      combineArgs: {
+        darkMode: [false, true],
+        baseFontSize: [13, 16],
```

Comment:
can we update these numbers and others in subsequent stories to use `BaseFontSize`?